packages/frontend/src/components/explore/content.js

```diff
@@ -0,0 +1,268 @@
+import antisocialApeClubIcon from './media/antisocialApeClub.png';
+import apollo42Icon from './media/apollo42.png';
+import arbitoorAggregatorIcon from './media/arbitoorAggregator.png';
+import armoredKingdomIcon from './media/armoredKingdom.png';
+import astrodaoIcon from './media/astrodao.png';
+import astroStakersIcon from './media/astroStakers.png';
+import burrowIcon from './media/burrow.png';
+import collectNFTsSectionIcon from './media/collectNFTsSectionIcon.svg';
+import cryptoheroeIcon from './media/cryptoheroe.png';
+import dragonflyCapitalPartnersIcon from './media/dragonflyCapitalPartners.png';
+import exchangeSectionIcon from './media/exchangeSectionIcon.svg';
+import galaxyOnlineIcon from './media/galaxyOnline.png';
+import hashRushIcon from './media/hashRush.png';
+import jumboExchangeIcon from './media/jumboExchange.png';
+import linearProtocolIcon from './media/linearProtocol.png';
+import metamonIcon from './media/metamon.png';
+import metapoolIcon from './media/metapool.png';
+import mintbaseIcon from './media/mintbase.png';
+import nearnautsIcon from './media/nearnauts.png';
+import parasIcon from './media/paras.png';
+import pembrockFinanceIcon from './media/pembrockFinance.png';
+import playToEarnSectionIcon from './media/playToEarnSectionIcon.svg';
+import rainbowbridgeIcon from './media/rainbowbridge.png';
+import refFinanceIcon from './media/refFinance.png';
+import spinIcon from './media/spin.png';
+import startEarningSectionIcon from './media/startEarningSectionIcon.svg';
+import sweatEconomyIcon from './media/sweatEconomy.png';
+import tamagoIcon from './media/tamago.png';
+import tonicDexIcon from './media/tonicDex.png';
+
+
+export const trandingProjects = [
+    {
+        icon: refFinanceIcon,
```

Comment:
For now Icons looks blurred on retina displays.
Lets create another set of icons (2x) and use them in case of high resolution displays.
See: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#resolution_switching_same_size_different_resolutions

Reply:
I know that's because of a png format. I'll ping design team to make something with it.